.github/workflows/sidecar-check.yml

```diff
@@ -72,4 +72,5 @@ jobs:
           git config --global user.name "Mykhailo Kuznietsov"
           git config --global user.email "mkuznets@redhat.com"
           export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
-          ./arbitrary-users-patch/check_sidecar_image_digests.sh
+          BUGFIX_BRANCH="$(git branch -a | grep "remotes/origin/7.*" | cut -d '/' -f 3 | sort --version-sort | tail -n1)"
```

Comment:
what if a topic branch is created called 7.32.x-some-reason? will that be used instead of 7.32.x ?

Reply:
right, I should add protection from that